main.js

```diff
@@ -0,0 +1,24 @@
+var data = {
+  location: {}
```

Comment:
Why is this an object? What do you intend to put in here? It doesn't seem to be updating.

Reply:
On the step below, I thought we were supposed to update the data model with the car's locatio:

_Add a location to your data model to hold the car's x and y coordinates on the page.__